src/org/omegat/Bundle.properties

```diff
@@ -2324,7 +2324,7 @@ GIT_ERROR=Git '{0}' error: {1}
 GIT_CONFLICT=Push failed. Will be synchronized next time.
 GIT_SAVE_PLAINPASSWORD=Allow to save password as plain text
 GIT_CREDENTIAL_MESSAGE=Git notification: {0}
-VCS_OFFLINE=No connection available, the project is in offline mode.
+VCS_OFFLINE=OmegaT does not have access to the remote repository: verify your connection or your credentials. The project is in offline mode.
```

Comment:
Please change key when meaning is changed.
It does not fix in localized message.